### ISSUE TITLE
Display Issue

### ISSUE THREAD

**Issue by JupiterPatel1** [open-learning-exchange/open-learning-exchange.github.io#3140]
When you click on the Contributors, you will get various frameworks and softwares instead of the contributors displayed on the page. 

![image](https://user-images.githubusercontent.com/59903633/153340652-2ef409a0-9da1-40e0-9d52-91ce16b01815.png)

Proposed Solution:-

1. Either, change the name of contributors to help 
2. Add the name of the contributors who are the members of the OLE Team

**Comment by lmmrssa** [comment]
@JupiterPatel1 please change it to help

**Comment by rheam97** [comment]
Could we maybe change the name to How to help? Just so it's explicit. Because help implies tech support or something like that.

**Comment by rlam20** [comment]
This could also be changed to 'Resources', 'Documentation', or 'Software Tutorials', as everything listed here relates to specific software that's used across the teams, and specific steps on how to use them.

**Comment by ewhang5** [comment]
I personally think that this section is a bit confusing. I don't think "Orientation" should be open to the general public and could maybe be sent out to those that are accepted individually? As for the other dropdown options, I think the title of "Projects" could work well. I'm not sure I understand why this would be labeled as "help" as these are more informative pages of what the current projects are.

**Comment by Kuljeet1998** [comment]
It still displays frameworks I believe. Let me if this needs to be taken up.

![Screenshot 2024-06-21 at 12 23 55 AM](https://github.com/open-learning-exchange/open-learning-exchange.github.io/assets/47841620/7af74f10-b1c5-4aac-bcb2-f8d649740828)

**Comment by deeppp15** [comment]
I think maybe we can add a heading of Projects in this section above the project names to provide more clarity. And we can also add the section of projects from VI Intern to the homepage showcasing active projects OLE is currently working on.

**Comment by huitk** [comment]
I think we should change the header to "Innovative Technologies".

**Comment by 3011089503** [comment]
I agree to change its name. Maybe "useful tools" are good. And I think the orientation can be a new header.
